corehq/tests/nose.py

```diff
@@ -192,6 +192,8 @@ def __init__(self, tests, runner):
         self.reuse_db = reuse_db
         self.skip_setup_for_reuse_db = reuse_db and reuse_db != "reset"
         self.skip_teardown_for_reuse_db = reuse_db and reuse_db != "teardown"
+        if reuse_db:
+            runner.keepdb = True
```

Comment:
Curious if and why this became necessary now? Should we have been doing this all along?

Reply:
I think we should have but it wasn't an option when this was first implemented. Setting this to True will prevent Django from trying to drop a DB if it already exists.